src/js/tabs/pid_tuning.js

```diff
@@ -2253,7 +2277,7 @@ TABS.pid_tuning.updateRatesLabels = function() {
             y -= (height/2); if(y<0) y=0; else if(y>context.height) y=context.height; // prevent balloon from going out of canvas
 
             // check that the balloon does not already overlap
-            for(var i=0; i<dirty.length; i++) {
+            for(const [i] of dirty.entries()) {
```

Comment:
Why use the index here, and not just iterate over the values? Also, keep in mind that `.entries()` returns an array of enumberable properties, and not of the iterator elements, so this is not equivalent.

Reply:
This article led me to use this after Sonar complaints:
https://flaviocopes.com/how-to-get-index-in-for-of-loop/